### PR TITLE
Show "Add Anime" at the bottom of the anime tables

### PR DESCRIPTION
**Pull request by MGaetan89**
The last line of the shows table include a "Add Show" link. If anime are separated from the other shows, the label is still "Add Show", instead of "Add Anime". This simply changes that :)

### CODE REVIEW

File: gui/slick/interfaces/default/home.mako [hunk 1]
```diff
@@ -612,7 +612,13 @@ $(document).ready(function(){
 
     <tfoot class="hidden-print">
         <tr>
-            <th rowspan="1" colspan="1" align="center"><a href="${sbRoot}/home/addShows/">Add Show</a></th>
```
Comment (OmgImAlexis) [inline gui/slick/interfaces/default/home.mako]: To make this fit with the rest of the file use this instead.

````
<th rowspan="1" colspan="1" align="center"><a href="${sbRoot}/home/addShows/">Add ${['Show', 'Anime](curListType == "Anime")}</a></th>
````

**Comment by miigotu** [comment]
@MGaetan89 yes, I would rather see it in the conforming style like @OmgImAlexis suggests.

**Comment by MGaetan89** [comment]
Of course. I have to try to turn the "Python" switch "on" in my brain when working on SickRage :P

**Comment by miigotu** [comment]
Squash pl0x

**Comment by MGaetan89** [comment]
Done

**Pull request merged**